crates/task-impls/src/events.rs

```diff
@@ -106,12 +106,14 @@ pub enum HotShotEvent<TYPES: NodeType> {
         VidCommitment,
         <TYPES::BlockPayload as BlockPayload>::Metadata,
         TYPES::Time,
+        FeeData<TYPES>,
     ),
     /// Event when the transactions task has sequenced transactions. Contains the encoded transactions, the metadata, and the view number
     BlockRecv(
         Vec<u8>,
         <TYPES::BlockPayload as BlockPayload>::Metadata,
         TYPES::Time,
+        FeeData<TYPES>,
```

Comment:
I don't think we should change anything right now, but I just want to note that these gradually-longer event types might start to become painful as time goes on, but I don't have a good solution yet.

Reply:
Agreed. I'd go further and say that we're overusing tuples, especially in enum variants, and propose something drastic like banning tuples with more than two fields or such. I'm going to raise this in today's Consensus meeting.